src/meta/model_v2/src/index.rs

```diff
@@ -27,7 +27,7 @@ pub struct Model {
     pub index_table_id: TableId,
     pub primary_table_id: TableId,
     pub index_items: ExprNodeArray,
-    pub original_columns: I32Array,
+    pub index_columns_len: u32,
```

Comment:
Could you also help to update the column definition here:

https://github.com/risingwavelabs/risingwave/blob/590fd7edf79c9fa06a91d0401ca3c34670506aa1/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L1087

https://github.com/risingwavelabs/risingwave/blob/590fd7edf79c9fa06a91d0401ca3c34670506aa1/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L672

Reply:
Sure. Let me hold this PR for a while. I just found that `include columns` should be included in indkey🤔.

Reply:
> indkey int2vector (references [pg_attribute](https://www.postgresql.org/docs/current/catalog-pg-attribute.html).attnum)

> This is an array of indnatts values that indicate which table columns this index indexes. For example, a value of 1 3 would mean that the first and the third table columns make up the index entries. Key columns come before non-key (included) columns. A zero in this array indicates that the corresponding index attribute is an expression over the table columns, rather than a simple column reference.

However, according to the PG official document, I think it should not include include columns.

Reply:
I think the current implementation makes more sense, so let's do it.